src/gerbil/prelude/core.ss

```diff
@@ -199,10 +197,9 @@ package: gerbil
     object? object-type
     struct-instance? class-instance?
     direct-instance?
-    make-object make-instance object-fill!
+    make-object object-fill!
```

Comment:
don't remove `make-instance`!!! It's a thing.

Reply:
undid.

Reply:
ok, fine, let it die -- I renamed it to `make-new-instance` to convey it is an instance without running the constructor.

Reply:
and the name `make-instance` replaces make-class-instance.